recipe/meta.yaml

```
@@ -10,31 +10,34 @@ source:
   sha256: fdc2739627eb1047c6b2df35045b654e07924fc9f6f8d490a34967365e75fd60
 
 build:
+  number: 1
   noarch: python
-  number: 0
   script:
     - {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
 
 requirements:
   host:
-    - python >=3.8
+    - python {{ python_min }}
     - pip
     - setuptools
     - hatchling >=1.5.0
```

Looks like `setuptools` has been superseded by `hatchling`. So think we can drop `setuptools`

```suggestion
    - hatchling >=1.5.0
```